test/src/utils.rs

```diff
@@ -119,6 +139,17 @@ where
     false
 }
 
+pub fn wait_get_blocks(secs: u64, net: &Net) -> bool {
+    wait_until(secs, || {
+        if let Ok((_, _, data)) = net.receive_timeout(Duration::from_secs(1)) {
+            if let Ok(message) = SyncMessage::from_slice(&data) {
+                return message.to_enum().item_name() == GetBlocks::NAME;
+            }
+        }
+        false
+    })
+}
+
```

Comment:
`wait_get_blocks` is not a general util function. What about moving it into `sync/utils.rs`?